samples/EverythingServer/SubscriptionMessageSender.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Extensions.Hosting;
+using ModelContextProtocol.Protocol.Types;
+using ModelContextProtocol.Server;
+using ModelContextProtocol.Protocol.Messages;
+
+internal class SubscriptionMessageSender(IMcpServer server, HashSet<string> subscriptions) : IHostedService
+{
+    public async Task StartAsync(CancellationToken cancellationToken)
+    {
+        while (!cancellationToken.IsCancellationRequested)
```

Comment:
This should also be a BackgroundService.

Reply:
Done